android/java/res/layout/fragment_setup_wallet.xml

```diff
@@ -15,80 +15,178 @@
+                    android:layout_alignParentTop="true"
+                    android:layout_alignParentStart="true"/>
+
+                <TextView
+                    android:id="@+id/setup_crypto_text_view"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_marginBottom="8dp"
+                    android:text="@string/setup_crypto"
+                    android:textColor="@color/wallet_text_color"
+                    android:textSize="22sp"
+                    android:lineSpacingExtra="2sp"
+                    android:layout_toEndOf="@+id/add_new_wallet_icon"
+                    style="@style/BraveWalletTextViewTitle"
+                    />
+
+                <TextView
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:text="@string/setup_crypto_description"
+                    android:textColor="@color/wallet_onboarding_secondary_text_color"
+                    android:textSize="14sp"
+                    android:lineSpacingExtra="4sp"
+                    android:layout_toEndOf="@+id/add_new_wallet_icon"
+                    android:layout_below="@+id/setup_crypto_text_view"
+                    />
+            </RelativeLayout>
+
+
+        </androidx.cardview.widget.CardView>
+
+        <androidx.cardview.widget.CardView
+            android:id="@+id/restore_wallet_card_view"
+            android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:padding="8dp"
-            android:background="?android:attr/selectableItemBackground"
-            android:layout_gravity="center_horizontal"
-            android:clickable="true"
-            android:focusable="true"
-            android:textAllCaps="false"
+            android:layout_marginHorizontal="32dp"
             android:layout_marginBottom="16dp"
-            android:text="@string/restore"
-            android:textSize="16sp"
-            android:textColor="@color/wallet_text_color"/>
+            android:foreground="?android:attr/selectableItemBackground"
+            android:clickable="true"
+            app:cardBackgroundColor="@color/brave_wallet_onboarding_card_view_background"
+            app:cardCornerRadius="10dp"
+            app:cardElevation="2dp"
+            app:cardUseCompatPadding="true">
+
+            <RelativeLayout
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:layout_margin="32dp">
+                <ImageView
+                    android:id="@+id/restore_wallet_icon"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:contentDescription="@null"
+                    android:src="@drawable/restore_wallet"
```

Comment:
I remember we had problems with `src` vector images on Android below 7 and we used `srcCompat`. From what I'm reading right now it also supports scaling. What do you think is it beneficial to use `srcCompat`. I would probably use it everywhere.

Reply:
`support-vector-drawable` library was fully integrated in Android 7.0 (API level 24): https://developer.android.com/develop/ui/views/graphics/vector-drawable-resources#vector-drawables-backward-solution.
The code used for rendering vector drawables from `src` and `srcCompat` should be unified now. 
Our minimum SDK is now Android Oreo (API 26), so we should be super safe here.
I also didn't notice any distortion, here is a screenshot:

![Screenshot 2024-02-05 at 5 48 22 PM](https://github.com/brave/brave-core/assets/3308503/63855288-e2c8-4b9f-a2c3-5ea8cdc26353)